auth_totp/models/res_users.py

```diff
@@ -13,6 +13,12 @@
 class ResUsers(models.Model):
     _inherit = 'res.users'
 
+    @classmethod
+    def _build_model(cls, pool, cr):
+        ModelCls = super(ResUsers, cls)._build_model(pool, cr)
+        ModelCls.SELF_WRITEABLE_FIELDS += ['mfa_enabled', 'authenticator_ids']
+        return ModelCls
+
```

Comment:
Is there some other mechanism for controlling access to a subset of a model's fields? The idea is that a user should be able to edit her email info or MFA authenticators but not other fields on her record, like the groups that she belongs to.